README.md

```diff
@@ -291,6 +291,12 @@ Emitted when a reconnect starts.
 
 Emitted after a disconnection.
 
+#### Event `'disconnect'`
+
+`function (packet) {}`
+
+Emitted after reciving disconnect packet from brocker. MQTT 5.0 feature.
```

Comment:
```suggestion
Emitted after receiving disconnect packet from broker. MQTT 5.0 feature.
```

Reply:
@mcollina, Disconnect packet from broker is MQTT 5.0 feature. I removed this.end(true) for working well logic of reconnects. In another way we must reconnect by code.